docker/library/api-scope-listing

```diff
@@ -1 +1 @@
+api:fhir api:oemr api:port launch/patient offline_access openid patient/AllergyIntolerance.read patient/Appointment.read patient/appointment.read patient/Binary.read patient/CarePlan.read patient/CareTeam.read patient/Condition.read patient/Coverage.read patient/Device.read patient/DiagnosticReport.read patient/DocumentReference.$docref patient/DocumentReference.read patient/Encounter.read patient/encounter.read patient/Goal.read patient/Immunization.read patient/Location.read patient/Medication.read patient/MedicationRequest.read patient/Observation.read patient/Organization.read patient/Patient.read patient/patient.read patient/Person.read patient/Practitioner.read patient/Procedure.read patient/Provenance.read user/allergy.read user/allergy.write user/AllergyIntolerance.read user/appointment.read user/appointment.write user/CarePlan.read user/CareTeam.read user/Condition.read user/Coverage.read user/dental_issue.read user/dental_issue.write user/Device.read user/DiagnosticReport.read user/document.read user/document.write user/DocumentReference.$docref user/DocumentReference.read user/drug.read user/Encounter.read user/encounter.read user/encounter.write user/facility.read user/facility.write user/Goal.read user/Immunization.read user/immunization.read user/insurance.read user/insurance.write user/insurance_company.read user/insurance_company.write user/insurance_type.read user/list.read user/Location.read user/medical_problem.read user/medical_problem.write user/Medication.read user/medication.read user/medication.write user/MedicationRequest.read user/message.write user/Observation.read user/Organization.read user/Organization.write user/Patient.read user/patient.read user/Patient.write user/patient.write user/Person.read user/Practitioner.read user/practitioner.read user/Practitioner.write user/practitioner.write user/PractitionerRole.read user/prescription.read user/Procedure.read user/procedure.read user/product.read user/Provenance.read user/soap_note.read user/soap_note.write user/surgery.read user/surgery.write user/transaction.read user/transaction.write user/user.read user/ValueSet.read user/vital.read user/vital.write
```

Comment:
I'm not sure about this change, btw as all new OAuth2 Clients creating with `/root/devtools register-oauth2-client` will include `user/user.read` scope